falcon/request.py

```diff
@@ -2436,7 +2436,7 @@ class RequestOptions:
     respectively.
     """
 
-    keep_black_qs_values: bool
+    keep_blank_qs_values: bool
```

Comment:
ok it was just the declaration, nothing was actually set on this variable so not a breaking change